docs/unittesting.md

```diff
@@ -88,13 +150,13 @@ Points to remember
 
 * A test contract cannot have a method with parameters. Having one such method will show error: `Method 'methodname' can not have parameters inside a test contract`
 * Number of test accounts are `3` before remix-ide release v0.10.0 and `10` afterwards
-* A test file which imports `remix_accounts.sol` might not compile successfully with `Solidity Compiler` plugin but it will work fine with Solidity Unit Testing plugin.
```

Comment:
While a test file which imports `remix_accounts.sol` might not compile successfully with `Solidity Compiler` plugin, **do not worry**, this will have no bearing on its success with Solidity Unit Testing plugin.

Reply:
done